src/HdInsightOnAks/HdInsightOnAks.Autorest/README.md

```diff
@@ -49,14 +46,12 @@ In this directory, run AutoRest:
 ### General settings
 > Values
 ``` yaml
-commit: f09aacf4c6b63be416212cb182f6b31e8bc6d545
+tag: package-preview-2023-11
```

Comment:
What is the use of the tag here?

Reply:
Our [Swagger](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md) has been updated to version package-2024-05. In this release, we use package-preview-2023-11.